Analytics/AnalyticsImplementing.swift

```diff
@@ -16,7 +16,7 @@ class AnalyticsImplementing: Analytics {
         self.analyticsLogger = analyticsLogger
     }
 
-    func track(_ event: AnalyticEvent, completion: @escaping (Result<Bool, Error>) -> Void) {
+    func track(_ event: AnalyticEvent, completion: @escaping EventLoggedCompletion) {
```

Comment:
Sorry, I was thinking it would be useful if we can parse the message to see why app logged event fails. But I didn't aware of the points you mentioned. I can close this PR, thank you!